label_studio/projects/models.py

```diff
@@ -740,11 +750,15 @@ def save(self, *args, update_fields=None, recalc=True, **kwargs):
             if update_fields is not None:
                 update_fields = {'control_weights'}.union(update_fields)
 
-        if self._label_config_has_changed():
-            self.__original_label_config = self.label_config
-
         super(Project, self).save(*args, update_fields=update_fields, **kwargs)
 
+        if label_config_has_changed:
+            # save the new label config for future comparison
+            self.__original_label_config = self.label_config
+            # if tasks are already imported, emit signal that project is configured and ready for labeling
+            if self.num_tasks > 0:
+                ProjectSignals.post_label_config_and_import_tasks.send(sender=Project, project=self)
```

Comment:
This call can affect performance and stability of project saving, I would call it async or at least add `try:except:`

Reply:
the performance overhead for handling this particular call is subtle. Although it makes sense to avoid any pitfalls in the corresponding handlers, I'll put checks there